security/pkg/server/ca/authenticate/kubeauth/kube_jwt.go

```diff
@@ -55,6 +56,7 @@ var _ security.Authenticator = &KubeJWTAuthenticator{}
 func NewKubeJWTAuthenticator(client kubernetes.Interface, clusterID string,
 	remoteKubeClientGetter RemoteKubeClientGetter,
 	trustDomain, jwtPolicy string) *KubeJWTAuthenticator {
+	log.Infof("XDS token type is: %v", security.XDSTokenType)
```

Comment:
Same as above, nor relevant in 99% of cases.

Reply:
Removed.